### PR TITLE
Migrate to Jekyll

### PR DESCRIPTION
**Pull request by gpotter2**
Hi !

This PR migrates the github pages to Jekyll, as proposed by  https://github.com/desowin/usbpcap/issues/58

It now uses markdown + a single theme. It'll be much easier to update something :-) You can chose to use a new theme, but for now it's the old one

You should probably check it carefully, to make sure it didn't break stuff.
Here's a live demo built with `bundle exec jekyll build`: it looks almost perfectly identical 

http://vmg.potter.fr/usbpcap/

Thanks again for your great work on usbpcap 😄

PS: the markdown was automatically generated using https://github.com/domchristie/turndown/pull/284, which I made specially for this, therefore some Markdown-specific things such as `quoting blocks` are not used, but probably should, especially for code.

### CODE REVIEW

File: captureformat.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Transfer-specific headers
+-------------------------
+
+All transfer-specific headers inherit the USBPCAP\_BUFFER\_PACKET\_HEADER, so first there is the USBPCAP\_BUFFER\_PACKET\_HEADER, then (if any) additional transfer-specific header data and then the transfer data.
+
+### USBPCAP\_TRANSFER\_ISOCHRONOUS
+
+When transfer is equal to USBPCAP\_TRANSFER\_ISOCHRONOUS (0) the header type is USBPCAP\_BUFFER\_ISOCH\_HEADER  
+
+/\* Note about isochronous packets:
```
Comment (desowin) [inline captureformat.md]: These two structures should probably also be wrapped inside code block.

File: captureformat.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+### USBPCAP\_TRANSFER\_INTERRUPT
+
+When transfer is equal to USBPCAP\_TRANSFER\_INTERRUPT (1) the header type is USBPCAP\_BUFFER\_PACKET\_HEADER  
+
+### USBPCAP\_TRANSFER\_CONTROL
+
+When transfer is equal to USBPCAP\_TRANSFER\_CONTROL (2) the header type is USBPCAP\_BUFFER\_CONTROL\_HEADER  
+
+#define USBPCAP\_CONTROL\_STAGE\_SETUP   0
```
Comment (desowin) [inline captureformat.md]: The defines and following structure should most likely be inside a code block.

File: captureformat.md (hunk 1 above)
Comment (gpotter2) [reply captureformat.md]: Done !

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+title: USBPcap
+email: your-email@example.com
```
Comment (gpotter2) [inline _config.yml]: ^

Comment (gpotter2) [reply _config.yml]: Feel free to use
```suggestion
suggestions
```
and merge it yourself :-)

**Comment by desowin** [comment]
I have merged this manually, removing the escapes inside the code blocks: 6acdff6e9353e271c9901a9046fdccbd72b196f0

**Pull request closed without merging**